kernel/src/actions/visitors.rs

```diff
@@ -211,20 +211,20 @@ impl RemoveVisitor {
 
         let size: Option<i64> = getters[5].get_opt(row_index, "remove.size")?;
 
-        // TODO(nick) stats are skipped in getters[6] and tags are skipped in getters[7]
+        // TODO(nick) tags are skipped in getters[6]
```

Comment:
Rescued comment: https://github.com/delta-incubator/delta-kernel-rs/pull/129/files#r1513701837

Yes, will have a follow-up to derive visitors.